lib/local-links-manager/import/csv_downloader.rb

```diff
@@ -5,7 +5,7 @@ class Error < RuntimeError; end
   class DownloadError < Error; end
   class MalformedCSVError < Error; end
 
-  def initialize(csv_url, header_conversions: {}, encoding: 'windows-1252')
+  def initialize(csv_url, header_conversions: {}, encoding: 'UTF-8')
```

Comment:
I think this makes the `LocalAuthoritiesURLImporter` csv import fail. You may have to update [this code](https://github.com/alphagov/local-links-manager/blob/master/lib/local-links-manager/import/local_authorities_url_importer.rb#L12) to be the following:
```
def initialize(csv_downloader = CsvDownloader.new(CSV_URL, encoding: 'windows-1252'))
```